components/alert/alert.js

```diff
@@ -171,6 +171,16 @@ class Alert extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		this.type = 'default';
 	}
 
+	static get demoProperties() {
+		return {
+			type: { type: 'select', options: ['default', 'critical', 'success', 'warning'], label: 'Alert Type' },
+			subtext: { type: 'text', label: 'Subtext' },
+			buttonText: { type: 'text', attribute: 'button-text', label: 'Button Text' },
+			hasCloseButton: { type: 'switch', attribute: 'has-close-button', label: 'Has Close Button' },
+			noPadding: { type: 'switch', attribute: 'no-padding', label: 'No Padding' }
+		};
+	}
+
```

Comment:
This is awesome!

I don't want to make it more complex but a future idea might be allowing you to see the different types within the same demo or something. I still think there is merit in certain properties being shown simultaneously for comparison purposes

Reply:
I also am still having internal debates on this being part of the component vs part of the demo

Reply:
Yeah, I had a similar thought as well, but had decided to make it internal in the end, just so that it was all in the same place. I'd initially added these fields to the actual properties, but decided to change that since doing it like this would also allow the user to choose the order of things rather than have them be alphabetical order or anything like that (since properties are usually in alphabetical order)

I'm not sure I understand what you mean by seeing the different types within the same demo though?

Reply:
Like I mentioned in our presentation, I think there's an argument to be made that showing varieties can have value sometimes, but specific _configurations_ should be demo-adjustable. 

For example, having a `click me to do x` button is utilitarian (Do you need this feature or not?), but deciding which type of alert, or if you need a normal vs subtle button, may be a softer problem that having choices on-screen at the same time can help solve.

With that in mind, something like this would be extremely cool:
![image](https://github.com/user-attachments/assets/1c7c0b2e-306c-4478-9aef-7c6729064a26)
But I don't think that's a problem we will solve for now :D